backend/api/Areas/Keycloak/Controllers/UserController.cs

```diff
@@ -34,10 +36,11 @@ public class UserController : ControllerBase
         /// </summary>
         /// <param name="keycloakService"></param>
         /// <param name="mapper"></param>
-        public UserController(IPimsKeycloakService keycloakService, IMapper mapper)
+        public UserController(IPimsKeycloakService keycloakService, IMapper mapper, IUserService userService)
         {
             _keycloakService = keycloakService;
             _mapper = mapper;
+            _userService = userService;
```

Comment:
Am I missing something? it seems like you are adding userService to this controller but then not using it.

Reply:
you are right, should be removed